tests/Examples/UnionExample.hs

```diff
@@ -17,13 +17,13 @@ type O2 = Object "O2" '[] '[Field "o2" Text]
 type T = Union "U" '[O1, O2]
 
 o1 :: Handler IO O1
-o1 = pure ((pure "hello from O1") :<> ())
+o1 = pure (pure "hello from O1")
 
 o2 :: Handler IO O2
-o2 = pure ((pure "hello from O2") :<> ())
+o2 = pure (pure "hello from O2")
 
 tHandler :: Handler IO T
-tHandler = o1 :<|> o2 :<|> ()
+tHandler = o1 :<|> o2
```

Comment:
Can you please add an example where you demonstrate that handlers are monoidal?

e.g. for `o1`, `o2`, `o3`, that:

```haskell
tHandler = (o1 :<|> o2) :<|> e3
```

and 

```haskell
tHandler = o1 :<|> (o2 :<|> o3)
```

Both compile?

No need to show they are equal.